### ISSUE TITLE
MacOS not working on Ventura update 13.0.1

### ISSUE THREAD

**Issue by NewMacusr** [1kc/razer-macos#720]
Just installed MacOS on M1 Pro with Ventura 13.0.1; had to allow application on Privacy & Security settings for it to load; However no devices are listed; cannot manage my keyboard using MacOS at the moment; Any ideas?

**Comment by CyanKnight12** [comment]
i am going through the same problem rn I don't know the solution

**Comment by InesAvdicZekic** [comment]
I am on MacOS Monterey 12.4 and I do not see the icon in the upper right corer/bar at all. I download it but after that nothing happens and iI am unable to add the device.

**Comment by CyanKnight12** [comment]
yeah same

**Comment by SScoorge** [comment]
did this issue fixed？i‘m using M2 mac and with the same issue

**Comment by CyanKnight12** [comment]
Yes it worked for me when i plugged in my peripherals. For my mouse, i plugged in the adaptor to my mac. Guys if you use wired peripherals it will work!

**Comment by NewMacusr** [comment]
Continue with same issue - wired or not. So far could not find a solution.

**Comment by torkat** [comment]
Similar issue here, I'm using M1 Air with Ventura. My DeathAdder is listed, however I can't change any color settings on it.

**Comment by ipoppo** [comment]
Add another case for my Kraken kitty v1 (usb) on M1 Pro / Ventura 13.1 is not working. Headset is listed but not responding to color settings.
